### ISSUE TITLE
Links in autogenerated data docs are either 404, or redirect to legacy.docs

### ISSUE THREAD

**Issue by mccalluc** [fivetran/great_expectations#3450]
**Describe the bug**
Doing the tutorial, I've generated data docs. On `file:///.../ge_tutorials/great_expectations/uncommitted/data_docs/local_site/index.html`, in the footer, I see:
>  To continue exploring Great Expectations check out one of these tutorials... 
```
<a href="https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations.html" class="btn btn-primary m-2" rel="noopener noreferrer" target="_blank">How to Create Expectations</a>
<a href="https://docs.greatexpectations.io/en/latest/guides/how_to_guides/validation.html" class="btn btn-primary m-2" rel="noopener noreferrer" target="_blank">How to Validate Data</a>
<a href="https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs.html" class="btn btn-primary m-2" rel="noopener noreferrer" target="_blank">How to Set Up a Team Site</a>
```

The first two links redirect with 301 status to legacy documentation; 
The last is a 404.
```
https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations.html
https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/validation.html
https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs.html
```

**To Reproduce**
1. Follow the tutorial.
2. Click on the links at the Bottom of the generated datadocs.

**Expected behavior**
Links should go to current documentation?

**Environment (please complete the following information):**
 - Operating System: MacOS
 - Great Expectations Version: 
   - superconductive/ge_tutorials @ 5db831a92af51f77dde19723a81a2cea92e2a756
   - great-expectations==0.13.35

**Additional context**
Mentioned this when talking with Ortal on September 24: Might already be filed or handled somewhere else.

**Comment by NathanFarmer** [comment]
Thanks for opening this issue @mccalluc! We are in the process of migrating the docs and this will be added to our queue.

**Comment by AFineDayFor** [comment]
Thank you @mccalluc, this was addressed in #4013
